lib/tasks/export_local_transactions.rake

```diff
@@ -0,0 +1,15 @@
+desc "Export Local Transactions' slugs, names, LGSL and optional LGIL codes as CSV"
+
+task :export_local_transactions => :environment do
+  require "csv"
+
+  csv_string = CSV.generate do |csv|
+    csv << ["slug","lgsl","lgil","title"]
+
+    Edition.where(_type: "LocalTransactionEdition").each do |lte|
+      csv << [lte.slug,lte.lgsl_code,lte.lgil_override,lte.title]
```

Comment:
I can't answer that as I don't know what the intended use of this CSV is.  I was just pointing out how that field works in case it might cause problems.